kobo/decorators.py

```diff
@@ -62,10 +62,10 @@ def new_func(*args, **kwargs):
             import datetime
             import kobo.shortcuts
             import kobo.tback
-            date = datetime.datetime.strftime(datetime.datetime.now(), "%F %R:%S")
-            data =  "--- TRACEBACK BEGIN: %s ---\n" % date
+            date = datetime.datetime.strftime(datetime.datetime.now(), "%F %R:%S").encode()
```

Comment:
Isn't an explicit encoding necessary here? For example, ``utf-8``.

Reply:
No, because `utf-8` is the default value for encoding.